pysph/tools/tests/test_dump_xdmf.py

```diff
@@ -0,0 +1,60 @@
+import shutil
+import unittest
+from pathlib import Path
+from tempfile import mkdtemp
+
+import numpy as np
+import vtk
```

Comment:
OK this is a very hard and difficult dependency and I would rather keep this as an optional one. Can you use a `pytest.importorskip` to avoid baking this into a hard dependency please?

Reply:
Yes, I will do this.